README.rst

```diff
@@ -88,3 +113,10 @@ More usage examples
 -------------------
 See the `documentation <https://pytorch-datastream.readthedocs.io/en/latest/>`_
 for more usage examples.
+
+
+Python 3.6 support
+==================
+Releases prior to version `0.3.3` are wrongly marked as supported by `Python 3` and will crash on import under `Python 3.6` and older.
```

Comment:
We can have this if you want but we don't have any users yet so this might be a source of confusion to people who don't read the entire text (most people)